plutus-exe/src/Main.hs

```diff
@@ -1,13 +1,16 @@
 {-# LANGUAGE FlexibleContexts #-}
 module Main (main) where
 
-import qualified Language.PlutusCore  as PC
+import qualified Language.PlutusCore                        as PC
```

Comment:
We generally call Plutus Core PLC. I'm not really sure whether it should be imported like that, but it seems less confusing to use the same abbreviation everywhere.

Reply:
Don't mind! I'll make it PLC